designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessDBQueryRepository.scala

```diff
@@ -112,7 +112,7 @@ object ProcessDBQueryRepository {
     processVersionId = actionData._1.processVersionId.getOrElse(throw new AssertionError(s"Process version not available for finished action: ${actionData._1}")),
     performedAt = actionData._1.performedAtTime.getOrElse(throw new AssertionError(s"PerformedAt not available for finished action: ${actionData._1}")),
     user = actionData._1.user,
-    action = actionData._1.action,
+    actionType = actionData._1.action,
```

Comment:
We should change action->actionType in DB as well

Reply:
Fixed.